serialization.md

```diff
@@ -697,17 +697,13 @@ Tree Root Helper Functions
 ### State Root
 
 ```k
-    syntax Map ::= "#precompiledContracts" [function]
- // -------------------------------------------------
-    rule #precompiledContracts
-      => #parseByteStackRaw( Hex2Raw( #unparseData( 1, 20 ) ) ) |-> #emptyContractRLP
-         #parseByteStackRaw( Hex2Raw( #unparseData( 2, 20 ) ) ) |-> #emptyContractRLP
-         #parseByteStackRaw( Hex2Raw( #unparseData( 3, 20 ) ) ) |-> #emptyContractRLP
-         #parseByteStackRaw( Hex2Raw( #unparseData( 4, 20 ) ) ) |-> #emptyContractRLP
-         #parseByteStackRaw( Hex2Raw( #unparseData( 5, 20 ) ) ) |-> #emptyContractRLP
-         #parseByteStackRaw( Hex2Raw( #unparseData( 6, 20 ) ) ) |-> #emptyContractRLP
-         #parseByteStackRaw( Hex2Raw( #unparseData( 7, 20 ) ) ) |-> #emptyContractRLP
-         #parseByteStackRaw( Hex2Raw( #unparseData( 8, 20 ) ) ) |-> #emptyContractRLP
+    syntax Map ::= #precompiledAccountsMap   ( Set )       [function]
+                 | #precompiledAccountsMapAux( List, Map ) [function]
+ // -----------------------------------------------------------------
+    rule #precompiledAccountsMap( ACCTS ) => #precompiledAccountsMapAux( Set2List( ACCTS ), .Map )
+
+    rule #precompiledAccountsMapAux( .List, M ) => M
+    rule #precompiledAccountsMapAux( (ListItem( ACCT ) => .List) _, (.Map => #parseByteStackRaw( Hex2Raw( #unparseData( ACCT, 20 ) ) ) |-> #emptyContractRLP) _ )
```

Comment:
We shouldn't be in the habit of doing map updates like this because it can crash when there are conflicts in the keys.

Instead do `MAP => MAP [ KEY <- VALUE ]`, which handles the case where the key doesn't exist or does exist already smoothly.